packages/cli/tests/create.test.js

```diff
@@ -19,6 +20,15 @@ describe('preact create', () => {
 		});
 	});
 
+	it(`Check if custom template is being used or not`, async () => {
```

Comment:
nit: it(`should use template.html from the github repo`)

Reply:
done